spec/controllers/idv/session_errors_controller_spec.rb

```diff
@@ -236,7 +236,7 @@
       let(:user) { create(:user) }
 
       before do
-        RateLimiter.new(rate_limit_type: :proof_address, user: user).increment_to_limited!
+        RateLimiter.new(rate_limit_type: :idv_resolution, user: user).increment_to_limited!
```

Comment:
Same as above, I think this was accidentally changed in https://github.com/18F/identity-idp/pull/5546